apps/sms/views/conversation/style/message.css

```diff
@@ -142,6 +142,20 @@
   color: #d1eeee;
 }
 
+.message.incoming .message-details .late-arrival-notice {
+  color: #026a69;
+  font-size: 1.3rem;
+  display: flex;
+  align-items: center;
+}
+
+.message.incoming .message-details .late-arrival-notice:before {
+  content: url('images/icons/late_clock.png');
```

Comment:
yes I confirm this; this is not high-dpi friendly.

to test this easily on Firefox, you can rename one of the high-dpi images (for example the @2x image) to the not-suffixed name.

I don't think we can make it work with a pseudo element, so better use the usual way of using background-image and background-size (or the background shortcut of course)